.github/workflows/main.yml

```
@@ -93,21 +93,28 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-
+      - name: Get dbt-core version
+        id: dbt-core-version
+        run: | 
+          echo "::set-output name=dbt-version::${{ 
+            ( contains( github.event.pull_request.base.ref, 'latest' ) || github.event.pull_request.base.ref == main) 
+            && github.event.pull_request.base.ref 
+            || main
```

```suggestion
            || 'main'
```
I think we can try quoting this